extension/resources/walkthrough/plots.md

```diff
@@ -1,61 +1,82 @@
 # Plots
 
-Select one or more experiments to visualize them in the
-[`Plots Dashboard`](command:dvc.showPlots). This is the extension's equivalent
-of the `dvc plots show` and `dvc plots diff` commands.
+Select one or more experiments to visualize in the
+[**Plots Dashboard**](command:dvc.showPlots).
 
-💡 If you don't have any DVC
-[plots](https://dvc.org/doc/command-reference/plots) in the project, start
-writing data points into JSON, YAML, CSV or TSV or saving plots as images
-(`.png`, etc) (check the [DVCLive](https://dvc.org/doc/dvclive) helper library
-if you use Python):
+> This is the similar to the [`dvc plots show`] and [`dvc plots diff`] commands.
+
+[`dvc plots show`]: https://dvc.org/doc/command-reference/plots/show
+[`dvc plots diff`]: https://dvc.org/doc/command-reference/plots/diff
+
+💡 To add [DVC plots] to the project, start writing data series into JSON, YAML,
+CSV, or TSV files; or save your own plot images (`.png`, etc.).
 
 <p align="center">
   <img src="images/plots-dump-with-open-file.png"
        alt="Code to Dump a JSON Plot File" />
-  <img src="images/plots-dump-with-dvclive.png"
-       alt="Code to Dump a JSON Plot File with DVCLive" />
   <img src="images/plots-dump-image.png"
        alt="Code to Dump an Image Plot File" />
 </p>
 
-Use `DVC: Show Plots` from the
-[Command Palette](command:workbench.action.quickOpen?%22>DVC:%20Show%20Plots%22)
-to open up the plots dashboard. The extension will display the following
-sections for selected experiments, that correspond to the different
-[types of plots](https://dvc.org/doc/command-reference/plots#supported-file-formats)
-supported by DVC:
+If you're using Python, the [DVCLive] helper library can save plots data for
```

Comment:
You are adding too much context to the page, too much structure. It tried as much as I can to keep it in a way that at least on my 13'' screen I can see some actual plots screenshots when open it

Reply:
The screenshots are great and look good on a small format which is also how I look at this when editing it:

<img width="1141" alt="image" src="https://user-images.githubusercontent.com/1477535/174653733-8702ff0b-f720-487c-b951-fcbe28814ada.png">

I didn't change any screenshot.

I separated the DVCLive part because it could be confusing mixed in the previous paragraph, since it only applies to data series (not images). It doesn't make the document any longer but again, it's not a strong opinion. I can put them all together again if you prefer.

Reply:
> too much structure

Are you referring to the H2s further down? Also not a strong opinion, I guess it's a stylistic choice. IMO it looks better and clearer (`# Plots > ## Plot types`) and again it doesn't make the page longer. Happy to roll back if you confirm.

Reply:
It's not about the page length per se. It's about when I can see any _plots_ on the page that is called "Plots". We want the screenshot with them visible asap. Adding more paragraphs, structure, etc are making it more and more like a tutorial. We should be throwing out things if possible, not adding them. It's similar to DVC's get started - we are fighting for every symbol here.

Reply:
> We should be throwing out things if possible... we are fighting for every symbol

OK, I compressed it again, in fact I merged a few things to make it even shorter. PTAL

> It's about when I can see any plots on the page that is called "Plots"

Maybe we should move the 💡 tip about setting up plots to the bottom then? Or just link to DVC docs